layouts/lodgingitems/lodgingitem.html

```diff
@@ -4,9 +4,12 @@
     </div>
 
     <div class="content">
-        <div class="category">
-            {{ partial "lodging/propertyCategory.html" .Params.property_category }}
-        </div>
+        {{ $propertyCategory := "func/GetLodgingCategory.html" .Params.property_category }}
```

Comment:
Not sure what would happen if `.Params.property_category` was not set, safer to wrap it in a with clause.
With the `with` shifting context, you don't even need any variable.

Also, Hugo will always assume `.html` when a partial’s extension is omitted, so for calling functions, I took the habit of omiting the extension (give them some uniqueness). Also, you can skip the partial and call them inside `with`:
```
{{ with .Params.property_category }}
  {{ with partial "func/GetLodgingCategory" . }}
    <div class="category">
      {{ . }}
    </div>
  {{ end }}
{{ end }}
```